how-to-docs/README.md

```diff
@@ -3,7 +3,7 @@ This folder contains a collection of how-to guides for both Developers and DevOp
 
 - Contents:
   - [Making an Artifactory robot](artifact-robot.md)
-  - [Getting a cert with letsencrypt and ingress](certs.md)
+  - [Getting a cert with letsencrypt and ingress](certificates.md)
```

Comment:
It's worth putting this in it's own commit, given it's not related to the Docker IP changes.